MREUnityRuntime/MREUnityRuntimeLib/Core/User.cs

```diff
@@ -18,6 +20,8 @@ internal class User : MixedRealityExtensionObject, IUser
 
         public IUserInfo UserInfo { get; private set; }
 
+        public UInt32 Groups { get; internal set; } = 1;
```

Comment:
What is the special meaning of group 1?

Reply:
It's the "default" group, that you're only in if you're not in any other groups. That relationship is maintained SDK-side, so no special logic in Unity.

---
In reply to: [267969008](https://github.com/Microsoft/mixed-reality-extension-unity/pull/60#discussion_r267969008) [](ancestors = 267969008)